en/docs/reference/pricing.md

```diff
@@ -2,229 +2,145 @@
-
-![Search for a flight](../assets/img/references/rest-api-example1.png)
-
-Let’s see how steps are calculated for the search process: 
-
-- As illustrated above, the client initiates a search for available flights through the web application. 
-- The web application then makes **1 API call to the [Search API](#searchAPI)** in Choreo. This is counted as **1 step**.
-- The search API in Choreo then makes **1 call to the Flights Database** to retrieve the information according to the search criteria entered by the client. This is counted as **1 step**. 
-
-- **Assuming the entire [execution time](#executionTime) for the search process takes less than 500ms** (starting from the web application making a call to Choreo, and Choreo making a call to the database), the search process thereby results in a **total of 2 steps**.
-
-<br/>
-
-#### Reserving a flight
-
-![Reserving a flight](../assets/img/references/rest-api-example2.png)
-
-Let’s see how steps are calculated for the reservation process:
-
-- As illustrated above, the client initiates the reservation process through the web application.
-- The web application proceeds to make **1 API call to the [Reservations API](#reservationsAPI)**. This is counted as 1 step.
-- The Reservation API then makes **1 API call to the [Bookings API](#bookingsAPI)**. This is counted as **1 step**. 
-- The bookings API makes **1 API call to the Booking Database** to update it with the booking details. This is counted as **1 step**.
-- Upon receiving the booking confirmation, the Reservation API makes **1 call to the [Twilio API](#twillioAPI)** to send an SMS confirmation to the customer. This is counted as **1 step**.
-
-- **Assuming the entire execution time for the reservation process takes 1200ms (1.2s)** , in addition to the above steps,  the total number of extra computation units consumed can be calculated as follows using ***floor((t)/500)) where t is the execution time***: 
-
-          floor(1200/500) = 2 units = 2 steps
-
-Therefore the total number of steps for the reservation process is: 
+## Plans
 
-           4 steps + 2 steps = 6 steps
+Choreo currently supports the following payment plans:
 
-We can calculate the **total number of steps for searching and reserving a flight** as follows:
+**Free**
 
-- **Search process**: Single execution of a search would cost a total of **2 steps**. 
-- **Reservation process**: Single execution of reservation process would cost a total of **6 steps**. 
-- **Total**: A single execution of the search and reservation process of a flight would cost an organization a total of **8 steps in total**.
+You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you want to deploy over 5 components.
 
-!!! note
-    The above cost calculation is dependent on the above architecture and design of the APIs.
-<br/>
+Choreo allocates up to a maximum of 2x vCPUs and 4 GB RAM infrastructure resources to your component workloads.
```

Comment:
Shouldn't we mention that we give 5 components in the free tier?

Reply:
Updated this section and mentioned that. Please check.